support/Environments/setup/frontera.sh

```diff
@@ -83,7 +93,7 @@ else
     tar -xzf libxsmm.tar.gz
     mv libxsmm-* libxsmm
     cd libxsmm
-    make CXX=g++ CC=gcc FC=gfortran -j4
+    make CXX="$compiler"++ CC="$compiler" FC=gfortran -j4
```

Comment:
doesn't this generate `gcc++` which isn't a thing?

Reply:
Oops, yup you're right. Fixed it by just specifying the `C` and `CXX` compilers individually.